### PR TITLE
add derivation extraction logic to getVarMD

### PR DESCRIPTION
**Pull request by stevegbrooks**
Hi,

I added some logic to the getVarMD function so that it extracts the derivation rules in the case that it is a derived variable. 

Thanks

### CODE REVIEW

**Comment by i-akiya** [comment]
I'll update the test that code and the documentations.

File: R4DSXML/R/getValMD.R
Comment (i-akiya) [inline R4DSXML/R/getValMD.R:1]: getValMD.R has been no changed except for code styling.

Comment (i-akiya) [reply R4DSXML/R/getValMD.R:1]: please remove getValMD.R from your pull request.

**Comment by stevegbrooks** [comment]
My apologies for the sloppy PR. I'll resubmit it soon. Thanks :)

**Pull request closed without merging**